src/applications/ivc-champva/10-10D/components/Applicant/applicantFileUpload.js

```diff
@@ -34,23 +35,29 @@ export const applicantBirthCertUploadUiSchema = {
   applicants: {
     'ui:options': { viewField: ApplicantField },
     items: {
-      ...titleUI('Upload birth certificate', ({ formData, formContext }) => (
-        <>
-          To help us process this application faster, submit a copy of{' '}
-          {applicantWording(
-            formData,
-            undefined,
-            true,
-            false,
-            formContext.pagePerItemIndex,
-          )}{' '}
-          birth certificate.
-          <br />
-          Submitting a copy can help us process this application faster.
-          <br />
-          {mailOrFaxLaterMsg}
-        </>
-      )),
+      ...titleUI(
+        ({ _formData, formContext }) =>
+          `Upload birth certificate ${isRequiredFile(formContext)}`,
```

Comment:
This adds "(required)" or "(optional)" to the header depending on if the page contains a file upload that is in the `requiredFiles` object. This change is applied to every file upload header in this file.